app/services/admin/query_all_commodities.rb

```diff
@@ -0,0 +1,19 @@
+module Admin
+  class QueryAllCommodities
+    def self.call(actual_date)
```

Comment:
Out of curiosity, what's the difference in performance now that we've removed the 0...9?

Reply:
Does the api return in < 60 seconds

Reply:
before - using 0..9% - about 45 sec,
now - using chapter short codes% - about 26 sec
I am not sure if the improvement is due only to that change (there might be other changes we've made)

Reply:
That's a good outcome. It must be that change :)